suites/compaction/rbd/rbd_mirror.yaml

```diff
@@ -0,0 +1,77 @@
+tests:
```

Comment:
please mention which conf to use to run this test suite

Reply:
done